api/pom.xml

```diff
@@ -57,6 +57,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-openapi</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Added this because the discussion we had yesterday about caching things. So I thought `v2` of the API should experiment with caching on the backend side too.

Maybe I should separate in another PR? I just this dependency and an annotation.

Reply:
Ah great, github only shows the code above the comment. I always add comments on top of what I want people to see.

Reply:
I think that we could keep it in this very same pull request.